src/impl/util.js

```diff
@@ -194,10 +194,11 @@ export function normalizeObject(obj, normalizer, ignoreUnknown = false) {
   for (const u in obj) {
     if (obj.hasOwnProperty(u)) {
       const v = obj[u];
-      if (v !== null && !isUndefined(v) && !Number.isNaN(v)) {
+      const numericValue = Number(v);
+      if (v !== null && !Number.isNaN(numericValue)) {
```

Comment:
@icambron, After changing this code, no invalid number string except (empty string or boolean value) pass the if condition. So if we test this function with this change for `DateTime.fromObject({ year: "hello" })` it will return the valid date-time with the value DateTime.Now because of the below line https://github.com/moment/luxon/blob/c16d7d14f6678886f50e3bb457c0b6e07f91470f/src/datetime.js#L646
Is it the correct behaviour??

Reply:
I think that is the correct behavior, yeah